CHANGELOG.md

```diff
@@ -31,3 +31,4 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Maintenance
 * Remove benchmarks folder from k-NN repo [#2127](https://github.com/opensearch-project/k-NN/pull/2127)
 ### Refactoring
+* Update unit test to include field with no live docs and minor refactor [#2167](https://github.com/opensearch-project/k-NN/pull/2167)
```

Comment:
I would reverse this to describe the minor refact and then say "and refactor some tests"

Reply:
Updated